CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 Mapbox welcomes participation and contributions from everyone.
 
 ## Unreleased
+#### Features
+
+#### Bug fixes and improvements
+- Fixed an issue where deprecated fields in `Road` type were always returning `null`s instead of the correct values. [#5396](https://github.com/mapbox/mapbox-navigation-android/pull/5396)
```

Comment:
maybe move notes to 2.2.0 release section, as changes were made specially for it?

Reply:
Actually we don't need this in the v2.2 changelog becuase it was introduced during pre-releases and now fixed, so no impact for v2.2.0 users. However, it has to remain in unreleased changes for `v2.3.0-alpha.2` because `v2.3.0-alpha.1` has this issue.